### DISCUSSION TITLE
Anzeige von 2 Neey Balancer in HomeAssitant

### DISCUSSION THREAD

**Discussion by adolf67** [shining-man/bsc_fw#193] [General]
Ist es möglich das man alle Werte von 2 Neey Balancer in Homeassitant über dem BSC anzeigen kann?

Ich vermute das nur die Werte immer nur von einem Neey Balancer angezeigt werden,wenn ich die Zellenspannungen vergleiche.

Ansonsten funktioniert die Software gut.

Ich wäre um eine Antwort dankbar.
Adi

**Comment by shining-man** [comment]
Du bekommst von allen verbunden NEEYs die Werte getrennt. Hast du einmal einen Screenshot von deinem Problem?

**Comment by tueftler0** [comment]
Ich denke das Problem liegt an dem YAML-File. Dieses musst du auch entsprechend des zweiten Neeys erweitern.
Schau mal im MQTT-Explorer, dort findest du den richtigen Pfad und solltest auch den zweiten Neey sehen.

**Comment by adolf67** [comment]
Guten Morgen an alle,

Ich habe 2 Yaml Dateien angelegt, für jeden NEEY Balancer eine Datei im Unterordener "Mqtts"

Muss ich für beide Neey Balancer nur eine Yaml Datei erstellen?

mfG
Adi

**Reply by tueftler0** [reply]
Nein, du kannst den Code ohne Probleme auch in mehreren Dateien getrennt unterbringen.
Ich denke einmal es liegt bei Dir wie gesagt am richtigen Pfad der Topics. UniqueIDs sind alle unterschiedlich?
Was sagt der MQTT-Explorer? Siehst du alle Neeys?

**Comment by adolf67** [comment]
Hallo!
Im  MQTTS- Explorer sehe ich alle Neeys mit bt 0 und bt 1und die Spannungswerte der einzelnen Zellen sind auch verschieden und es wird auch brav aktualisiert.

Nur unter "Geräte und Dienste" im Homeassitant unter MQTTS  sind die Werte für beiden Neeys die gleichen, das ist das komische wenn ich dort draufschaue.

Danke, erstmal für die Hilfe.

**Reply by adolf67** [reply]
Alle UniqueIDs sind verschieden.

**Reply by tueftler0** [reply]
Wenn sie im Explorer korrekt dargestellt werden, im Gerät selbst aber nicht, sollte der verwendete Topic-Pfad in einer der YAMLs nicht stimmen.
Daten kommen ja alle korrekt an.
Wenn du es nicht raus findest, bitte kleines Bild beider YAMLs hier einfügen.

**Reply by adolf67** [reply]
Hallo tueftler!
Danke für deinen Ratschlag ich habe den verwendeten Topic-Pfad geändert .
Jetzt funktioniert alles,danke.
Gruss
Adi

**Comment by tueftler0** [comment]
Schön dass es nun funktioniert.
Dann bitte das Issue schließen.
